lib/fs_utils/pipeline.js

```diff
@@ -65,6 +65,33 @@ const getDependencies = (data, path, compilerDeps, compiler) => {
   }
 };
 
+const compileStatic = (path, compiler) => {
+  let source;
+  return readFromCache(path).then(_source => {
+    source = _source;
+    return processJob(CompileStaticJob, {path, source, plugin: compiler});
+  }).then(data => {
+    return Object.assign({}, data, {source});
+  });
+};
```

Comment:
We don't need the regular pipeline because there are no linters and only *one* compiler. Also no deppack checks, etc